content/blog/2020-05-04-dvc-3-years-and-1-0-release.md

```diff
@@ -0,0 +1,198 @@
+---
+title: DVC 3 Years Anniversary and 1.0 Pre-release
```

Comment:
can we omit Anniversary - to make it fit into a single line

Reply:
Maybe "DVC Year 3 and 1.0 Pre-release"

Reply:
`DVC 3 Years and 1.0 Pre-release` ?

Reply:
`DVC 3 Years 🎉 and 1.0 Pre-release 🚀` ?

Reply:
What about `3 years of DVC and 1.0 Pre-release`?